test/test_helper.rb

```diff
@@ -14,4 +14,20 @@ class Sablon::TestCase < MiniTest::Test
   def teardown
     super
   end
+
+  class UIDTestGenerator
+    def initialize
+      @current_id = rand(10...100)
```

Comment:
This was a good solution to the random ID generation. Although you might want to fix the initial ID value (i.e. like  `@current_id = 1234`) otherwise the html insertion integration test will still fail even though we have a predictable next uuid value because we can't modify the "expected" word ml prior to the equality assertion like you do in converter_test.rb.

Reply:
Yeah I just did that and tried integration test but there was a problem that I need your opinion for, I tried to create a hyperlink using microsoft office 2016 in html_sample.docx file so I can compare it with the generated file, but apparently it adds more attributes and tags to the file document.xml that is not added by sablon generator, my guess is new features supported by office 2016. So I unzipped the html_sample.docx and added the hyperlinks manually, I am not sure if this is a valid way to test it, so I need your opinion in that matter.

Thanks

Reply:
The way I usually refresh the sample document is generate it straight from sablon from the template. Once the output looks and works like I expect it to then I commit the new docx sample and push. Occasionally, I'll unzip it and double check the contents if I want to be extra sure.

I don't actually open and edit the sample doc because like you said MS Word will pack a bunch of extra junk in there such as save and edit information into the XML.

Reply:
Well in this case I have generated a file with sablon and double checked it, it contains three hyperlinks; one normal, one bold and one italic then I replaced the html_sample.docx file with it. I think I have finished all of the reviews please check again and if there is no other issues I guess we are ready for the merge.

Thanks